### DISCUSSION TITLE
TreeSelect 输入的可以用逗号分隔筛选多个选项吗？

### DISCUSSION THREAD

**Discussion by Yucodes** [ant-design/ant-design#41200] [General]
![image](https://user-images.githubusercontent.com/92520987/224667538-cd42ee94-d9b3-45c9-a8b1-e44da6d7135d.png)
比如输入了西青,南开
TreeNode里只显示西青和南开字样的选项，其它未包含的选项不显示
现在单个输入可以用filterTreeNode 返回true
逗号分开的多个字符怎么实现？

**Comment by Yuiai01** [comment]
或许可以使用 `onSearch` 方法，对输入内容做特殊处理，然后自行过滤数据
